config/locales/controllers/admin/fr.yml

```
@@ -3,3 +3,5 @@ fr:
   admin:
     extensions_controller:
       update_success: Les extensions ont été mises à jour.
+    spaces_controller:
+      update_success: Space has been updated.
```

```suggestion
      update_success: L'espace a été mis à jour.
```

Maybe? I've used google translate before and confirmed that the translation was okay with french speaking team members. Hnece google translate seemed safe enough to use for short sentences.